wallet/src/actors/worker/methods.rs

```diff
@@ -135,6 +136,47 @@ impl Worker {
                 (id, default_account, Some(master_key))
             }
         };
+
+        let superblock_interval: i64 = 21;
```

Comment:
Magic number. Ideally this should be a constant somewhere out of this function, and have some documentation on what it means and where did its value come from.

Reply:
`2 * consensus_constants.superblock_period + 1`